Tasks/UniversalPackagesV0/task.json

```diff
@@ -316,6 +316,15 @@
             "required": "false",
             "groupName": "packagePublishDetails"
         },
+        {
+            "name": "publishedPackageVar",
+            "type": "string",
+            "label": "Package Ouput Variable",
+            "required": false,
+            "defaultValue": "",
+            "groupName": "packagePublishDetails",
```

Comment:
@mitchdenny - in the publish details group, not an output group like your other example?

Reply:
Updated to include an output group